docs(concepts): Describe audit trail feature (#5747)
zitadel/zitadel: docs/docs/concepts/features/audit-trail.md

## Accessing the Audit Log

### Last changes of an object

You can check the last changes of most objects in the [Console](/docs/guides/manage/console/overview).
In the following screenshot you can see an example of last changes on an [user](/docs/guides/manage/console/users).
The same view is available on several other objects such as organization or project.

![Profile Self Manage](/img/guides/console/myprofile.png)

### Event viewer

Administrators can see all events across an instance and filter them directly in [Console](/docs/guides/manage/console/overview).
Go to your instance settings and then click on the Tab **Events** to open the Event Viewer or browse to $YOUR_DOMAIN/ui/console/events  

![Event viewer](/img/concepts/audit-trail/event-viewer.png)

### Event API

Since everything that is available in Console can also be called with our APIs, you can access all events and audit data trough our APIs:

- [Event API Guide](/docs/guides/integrate/event-api)
- [API Documentation](/docs/category/apis/admin/events)

> Access to the API is possible with a Service User account, allowing you to integrate the events with your own business logic.

Author: added

## Using logs in external systems

You can use the [Event API](#event-api) to pull data and ingest it in an external system.

[Actions](actions.md) can be used to write events to the stdout and [process the events as logs](../../self-hosting/manage/production#logging).
Please refer to the zitadel/actions repository for a [code sample](https://github.com/zitadel/actions/blob/main/examples/post_auth_log.js).
You can use your log processing pipeline to parse and ingest the events in your favorite analytics tool.

> Theoretically it would be possible to send events directly with an http request to an external tool.

Reviewer: ```suggestion
It is possible to send events directly with an http request to an external tool.
```
